nautobot_device_lifecycle_mgmt/templates/nautobot_device_lifecycle_mgmt/inc/software_and_validatedsoftware_info.html

```diff
@@ -0,0 +1,11 @@
+{% load helpers %}
+{% if perms.dcim.view_softwareversion %}
+    {% if object.software_version %}
+        {% include 'nautobot_device_lifecycle_mgmt/inc/software_info.html' %}
+    {% endif %}
+{% endif %}
+{% if perms.nautobot_device_lifecycle_mgmt.view_validatedsoftwarelcm %}
+    {% if validsoft_table %}
```

Comment:
Both of these appear to be in the included template so can probably be removed here?

Reply:
Makes sense, left the ones in the included template.